README.md

```diff
@@ -177,27 +192,26 @@ After running `package.sh`, you should have a signed and notarized dmg file in t
 TODO
 
 # Debugging a HARP Build
-## Mac
-1. Download [Visual Studio Code for macOS](https://code.visualstudio.com/).
+## Visual Studio Code
+1. Download [Visual Studio Code](https://code.visualstudio.com/).
 2. Install the C/C++ extension from Microsoft.
 3. Open the _Run and Debug_ tab in VS Code and click _create a launch.json file_ using _CMake Debugger_.
 4. Create a configuration to attach to the process (see the following example code to be placed in `launch.json`).
 
-```
+```json
 {
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "(lldb) Launch",
+            "name": "(lldb) Standalone",
             "type": "cppdbg",
             "request": "launch",
-            "program": "${workspaceFolder}/build/HARP_artefacts/Debug/HARP.app",
-            "args": ["./test.wav"], // TODO - remove?
-            "stopAtEntry": false,
+            "program": "${workspaceFolder}/build/HARP_artefacts/Debug/HARP.app", // for macOS
```

Comment:
Comments look perfectly fine in VS Code. I'm not sure why they are like this here.